contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Understanding page load time metrics in performance monitoring
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-12
+featuredImage: todo
+tags: []
+---
+
+Waiting for slow websites is like watching paint dry. It's the bane of productivity, the destroyer of efficiency, and the enemy of progress.
+
+In this tutorial, we'll show you how to understand different metrics on how quickly your pages load and how to identify opportunities for improvement using Posthog's [performance monitoring in session recordings tool](https://posthog.com/blog/posthog-changelog#performance-monitoring-in-session-recordings).
+
+## Setup
+
+First, ensure that session recordings are [enabled]((https://posthog.com/docs/session-replay/manual)). Next, navigate to the Recordings tab on your Posthog Dashboard. Finally, click on the Network tab in the sidebar next to or below the video.
```

Comment:
I'd like to see a little more detail here so the reader doesn't have to go find it, such as:

- You must use our posthog-js library to enable recordings
- If using our JavaScript snippet, you can enable it in project settings (screenshot below you can use)

![Screenshot 2023-04-05 at 15 33 28](https://user-images.githubusercontent.com/92976667/230114204-16c1a189-aabb-4832-9d5a-43dc1c31d2d2.png)

Reply:
Additional screenshot for how to find project settings (optional)

![Screenshot 2023-04-05 at 15 42 34](https://user-images.githubusercontent.com/92976667/230116479-b4982ab2-5a24-44d4-ac3e-9c0471d70d6a.png)

Reply:
I think it is good how it is now. A quick mention about enabling in settings and installing the JavaScript SDK is good enough.